packages/errors/test/deep-send.test.js

```diff
@@ -0,0 +1,45 @@
+// This file is not really useful as an
```

Comment:
Not really a new file. Rather, migrated from eventual-send/test, but somehow github didn't notice.